content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,205 @@
+
+These rules add only leads with an associated contact method to the report.
+
+.. _crm/track_leads/active-status:
+
+Active status
+~~~~~~~~~~~~~
+
+Click the :icon:`fa-sitemap` :guilabel:`(Branch)` icon to the right of the `Phone is set` line, to
+add a new rule that branches from the rules above.
+
+Two horizontal sets of fields appear below a line showing :guilabel:`any of:` with a
+:icon:`fa-caret-down` :guilabel:`(down caret)` icon. This setting filters for
+records that match **any** of the rules contained inside. This uses the same logic as an OR (|)
+logical operator.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is set` in the next field.
+
+Next, click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is set` to
+create a new line of fields beneath it.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is not set` in the next field.
+
+.. image:: lead_distribution_report/active-set.png
+  :align: center
+  :alt: Use Active to include active status in the report.
+
+This rule adds the activity status of the lead to the report.
+
+.. note::
+  Active status is an important filter to include when creating a lead distribution report because
+  it includes **all** leads regardless of won/lost or active/inactive status in the report. This
+  provides a comprehensive view of all the leads assigned to each sales member.
+
+Group by
+~~~~~~~~
+
+Once all filters are set, click the :guilabel:`Add` button to add these filters to the search bar.
+To have the report grouped appropriately, click the :icon:`fa-caret-down` :guilabel:`(down caret)`
+icon, to the right of the search bar, and click :guilabel:`Salesperson` in the :guilabel:`Group
+By` section. All results are now grouped by the salesperson assigned to each lead.
+
+Once the rules for the filter are set, click the purple :guilabel:`Confirm` button at the bottom of
+the pop-up menu to save the custom filter and close the pop-up menu.
+
+The :guilabel:`Pipeline Analysis` dashboard is now displayed again with each filter rule in the
+search bar.
+
+Click the :icon:`fa-area-chart` :guilabel:`(Graph)` icon, to the right of the
+search bar, to view the report as a bar chart. Alternatively, click the :icon:`oi-view-list`
+:guilabel:`(List)` icon to view leads in a grouped list.
+
+.. tip::
+  To save the filter so it can easily be re-applied, click the :guilabel:`Save current search`
+  button in the :guilabel:`Favorites` section of the search bar drop-down menu.
+
+  Next, type a name for the filter in the text box below. Check the :guilabel:`Shared` checkbox to
+  have the filter shared with any user with access to the pipeline. Finally, click the purple
+  :guilabel:`Save` button below to save the filter.
+
+  The filter will now appear with the name it was given under the :guilabel:`Favorites` section of
+  the drop-down menu and can be re-applied by clicking on it.
```

Comment:
Alignment
```suggestion
.. tip::
   To save the filter so it can easily be re-applied, click the :guilabel:`Save current search`
   button in the :guilabel:`Favorites` section of the search bar drop-down menu.

   Next, type a name for the filter in the text box below. Check the :guilabel:`Shared` checkbox to
   have the filter shared with any user with access to the pipeline. Finally, click the purple
   :guilabel:`Save` button below to save the filter.

   The filter will now appear with the name it was given under the :guilabel:`Favorites` section of
   the drop-down menu and can be re-applied by clicking on it.
```